concent_api/core/tests/test_auth_report_computed_task.py

```diff
@@ -8,12 +7,13 @@
 from golem_messages         import load
 from golem_messages         import message
```

Comment:
Please adapt all imports to our coding style in this file

Reply:
:+1: done